pytket/ruff.toml

```diff
@@ -0,0 +1,10 @@
+exclude = ["pytket/_tket"]
```

Comment:
It might be good to put in the select section in here, too. Similar to https://github.com/CQCL/pytket-quantinuum/blob/main/ruff.toml ?

Reply:
Thanks, I didn't realize that there are a lot of useful checks that are off by default. I'll see how many I can easily add.